lib_eio/unix/fork_action.c

```diff
@@ -67,13 +70,17 @@ CAMLprim value eio_unix_fork_execve(value v_unit) {
 }
 
 static void action_fchdir(int errors, value v_config) {
+  #ifdef _WIN32
+  uerror("Unsupported operation on windows", Nothing);
```

Comment:
We can't allocate OCaml exceptions here (or in the other actions) as we're in the child and anything involving GC isn't going to work.

```suggestion
  eio_unix_fork_error(errors, "action_fchdir", "Unsupported operation on windows");
  _exit(1);
```